src/settings-ui/Settings.UI/Views/GeneralPage.xaml

```diff
@@ -207,26 +208,555 @@
                     <controls:Setting x:Uid="ColorModeHeader" Icon="&#xE790;">
-                                <controls:Setting.Description>
-                                    <HyperlinkButton Click="OpenColorsSettings_Click"
+                        <controls:Setting.Description>
+                            <HyperlinkButton Click="OpenColorsSettings_Click"
                                                      x:Uid="Windows_Color_Settings"/>
+                        </controls:Setting.Description>
+                        <controls:Setting.ActionContent>
+                            <ComboBox SelectedIndex="{x:Bind Mode=TwoWay, Path=ViewModel.ThemeIndex}" MinWidth="{StaticResource SettingActionControlMinWidth}">
+                                <ComboBoxItem x:Uid="Radio_Theme_Dark"/>
+                                <ComboBoxItem x:Uid="Radio_Theme_Light"/>
+                                <ComboBoxItem x:Uid="Radio_Theme_Default"/>
+                            </ComboBox>
+                        </controls:Setting.ActionContent>
+                    </controls:Setting>
+                    <controls:Setting x:Uid="GeneralPage_RunAtStartUp">
+                        <controls:Setting.ActionContent>
+                            <ToggleSwitch IsOn="{Binding Mode=TwoWay, Path=Startup}" x:Uid="ToggleSwitch"/>
+                        </controls:Setting.ActionContent>
+                    </controls:Setting>
+                </controls:SettingsGroup>
+
+                <!-- Start General_SettingsBackupAndRestore NEW -->
+
+                <controls:SettingsGroup x:Uid="General_SettingsBackupAndRestore" Visibility="Visible">
+                    <controls:SettingExpander IsExpanded="True">
+                        <controls:SettingExpander.Header>
+                            <controls:Setting x:Uid="General_SettingsBackupAndRestore" Icon="&#xE777;" Style="{StaticResource ExpanderHeaderSettingStyle}">
+                                <controls:Setting.Description>
+                                    <TextBlock Style="{StaticResource SecondaryTextStyle}">
+                                                <Run x:Uid="General_BackupAndRestoreDescription" />
+                                    </TextBlock>
                                 </controls:Setting.Description>
                                 <controls:Setting.ActionContent>
-                                    <ComboBox SelectedIndex="{x:Bind Mode=TwoWay, Path=ViewModel.ThemeIndex}" MinWidth="{StaticResource SettingActionControlMinWidth}">
-                                        <ComboBoxItem x:Uid="Radio_Theme_Dark"/>
-                                        <ComboBoxItem x:Uid="Radio_Theme_Light"/>
-                                        <ComboBoxItem x:Uid="Radio_Theme_Default"/>
-                                    </ComboBox>
+
+
+                                    <Grid Visibility="Visible">
```

Comment:
Instead of using a Grid, I'd recommend using a StackPanel instead and setting its Orientation to Horizontal. You can set the Spacing to 4 so there's a small gap in between the two buttons :)

Reply:
WPF/XAML takes me a while to get "right" and there are many ways to do the same thing. I think "could be better" can wait, and perhaps done by those like you which are clearly better at it than me.